packages/functions/src/donor/CancelAppointmentHandler.ts

```diff
@@ -34,7 +34,8 @@ export default async function (
     console.error("Error notifying on cancelled appointment", appointment.id, e)
   );
 
-  const updatedAppointment = removeDonorFromDbAppointment(appointment);
+  const updatedAppointment =
+    DbAppointmentUtils.removeDonorFromDbAppointment(appointment);
```

Comment:
Doesn't this negate the concept of working with a DAL?

Reply:
I don't understand why this should be in the DAL.

It is not reading or writing any data, just mutating an object.

This is a classic util.

What do you think? Should this be in the DAL? Why?

Reply:
Mutating is CRUD, I understand DAL === CRUD?

If there should be any complex permission handling for specific mutating operations,
they should be handled directly and not in a util file, no?

Reply:
> Mutating is CRUD, I understand DAL === CRUD?

🤷 

> If there should be any complex permission handling

What permissions handling?